src/qibolab/instruments/qblox/acquisition.py

```diff
@@ -3,7 +3,7 @@
 
 import numpy as np
 
-SAMPLING_RATE = 1e9
+SAMPLING_RATE = 1
```

Comment:
If we are downscaling it some of Zurich devices would have float as sampling rates

Reply:
> If we are downscaling it some of Zurich devices would have float as sampling rates

I pointed out both because, to me, they are two separate problems:
- it could be float in general, because of rates <Gsps, but also because it might be higher but non-integer
- if downscaling is possible, then the value is not constant per-driver (so, we need a different mechanism)

Reply:
I don’t see any issue with using non-integer sampling rate, in fact RFSoc is already using non-integer. The only potential issue is that the waveform envelop generation in pulses.py does some roundings to calculate the number of samples, but this could be an issue for integer sampling rates as well.

Regarding the comment, I was a bit relaxed with the type since Python is usually automatically casting ints to floats if needed, but I can convert everything to float.

For downsampling, I don’t know if it is supported by the instruments.

Reply:
It's supported by Zurich, I don't think we will want to see it now at least. https://docs.zhinst.com/hdawg_user_manual/functional_description/specific/awg.html?h=sampling#constants-and-variables

Reply:
Ok, thanks @Jacfomg at least we know it's generally possible. Now we're left with the only doubt whether it is potentially useful or not at all.
I would ask the lab. If we can not imagine any reasonable use case, let's keep it constant, and upgrade when needed.

> Regarding the comment, I was a bit relaxed with the type since Python is usually automatically casting ints to floats if needed, but I can convert everything to float.

Indeed, it is not required. But I'd try to be consistent (for as long as it doesn't cost much more than a couple of characters here and there)

Reply:
The only use I'm aware of and I think Zurich may be using it's related to having devices with similar sampling_rates and the one with the higher one may be downscaled for some synchronization issues.

Reply:
Then there are three possibilities:
1. Hardcode the value in the instruments. It is possible to hardcode different values for different instruments in the same driver (eg. Zurich) but in that case care should be taken for what is used where (in the driver).
2. Pass the value to the instrument during initialization in the `create`.
3. Pass the value as platform setting in the YAML.

3 is my least favorite as I think the sampling rate is static (not expected to change during calibration) so I would not put it in the YAML. Other than that I don't have strong opinion. 1 is easier for the user but less flexible.

Reply:
I would have not considered 3 at all, following our current separation (as you said, it's not going to be calibrated).

The doubt was whether 1 is enough or we need 2. If it's enough, better 1.